znflow/graph.py

```diff
@@ -47,13 +47,21 @@ def default(self, value, **kwargs):
 
 
 class DiGraph(nx.MultiDiGraph):
+    def __init__(self, *args, disable=False, **kwargs):
+        self.disable = disable
+        super().__init__(*args, **kwargs)
+
     def __enter__(self):
+        if self.disable:
+            return self
```

Comment:
When you call a contextmanager:
```
with znflow.DiGraph() ..
```
it will call the `__enter__` method upon opening it and the `__exit__` method when the context manager is closed again.